services/workflows-service/scripts/alerts/generate-alerts.ts

```diff
@@ -119,7 +132,7 @@ export const ALERT_INLINE_RULES = [
         havingAggregate: AggregateType.COUNT,
 
         direction: 'inbound',
-        excludedCounterpartyIds: ['9999999999999999', '999999******9999'],
+        // excludedCounterparty: ['9999999999999999', '999999******9999'],
```

Comment:
Same

Reply:
we need to modify this exclusion from other rules.
so once i changed the type i had to take care for it cross all other rules